File: docs/components_page/components/dropdown/content.py
import dash_bootstrap_components as dbc
import dash_html_components as html

dropdown = dbc.DropdownMenu(
    [
        dbc.DropdownMenuItem("Header", header=True),
        dbc.DropdownMenuItem("An item"),
        dbc.DropdownMenuItem(divider=True),
        dbc.DropdownMenuItem("Active and disabled", header=True),
        dbc.DropdownMenuItem("Active item", active=True),
        dbc.DropdownMenuItem("Disabled item", disabled=True),
        dbc.DropdownMenuItem(divider=True),
        html.P(
            "You can have other content here like text if you like.",
            className="text-muted px-4 mt-4",
        ),
    ],
    label="Menu",
)

File: components_page/components/dropdown/content.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

dropdown_content <- dbcDropdownMenu(
  list(
    dbcDropdownMenuItem("Header", header = TRUE),
    dbcDropdownMenuItem("An item"),
    dbcDropdownMenuItem(divider = TRUE),
    dbcDropdownMenuItem("Active and disabled", header = TRUE),
    dbcDropdownMenuItem("Active item", active = TRUE),
    dbcDropdownMenuItem("Disabled item", disabled = TRUE),
    dbcDropdownMenuItem(divider = TRUE),
    htmlP(
      "You can have other content here like text if you like.",
      className = "text-muted px-4 mt-4"
    )
  ),
  label = "Menu"
)
